h5py/api_types_hdf5.pxd

```diff
@@ -832,7 +833,39 @@ cdef extern from "hdf5.h":
   ctypedef herr_t (*H5A_operator2_t)(hid_t location_id, char *attr_name,
           H5A_info_t *ainfo, void *op_data) except 2
 
-
+# === H5FD - Subfiling ========================================================
```

Comment:
Please add to the existing H5FD block (above H5G) rather tan creating a new one.